ros/install.bash

```diff
@@ -3,7 +3,14 @@
 
 if [ -z "$TUE_ROS_DISTRO" ]
 then
-    tue-install-error "TUE_ROS_DISTRO was not set"
+    echo "[tue ros install] TUE_ROS_DISTRO was not set"
+    echo "[tue ros install] Did you run the bootstrap-ros1.bash script?"
+    echo "[tue ros install] You can also set it manually like:"
+    echo ""
+    echo 'mkdir -p "${TUE_ENV_DIR}/.env/setup"'
+    echo 'echo "export TUE_ROS_DISTRO=melodic" > "${TUE_ENV_DIR}/.env/setup/user_setup.bash"'
```

Comment:
Melodic specific

Reply:
It's a hint. We can't do much better in the hint because variables are wrong?